### DISCUSSION TITLE
Have a feature to inform users about Traffic

### DISCUSSION THREAD

**Discussion by Aryan-Sagavekar** [organicmaps/organicmaps#4889] [Ideas]
@biodranik and @rtsisyk 
I saw the livestream you had on YouTube talking about OM. There was an issue discussed there about the real time traffic updates. 
As we know OM does not track anyone so it would be hard to provide real time traffic data but what can be done is we can encourage user to provide some information about traffic heavy roads through some UI and code. We could then inform other users based off of that information like "This road is usually busy" or something like that. I am currently a bit unclear on the details as I haven't researched that much yet, but I think this could be a really interesting feature.
This is just a suggestion and I would like to know your views about it (Also, should I create an issue?).

**Comment by biodranik** [comment]
Aren't there already some related issues? Manual adding of road issues is similar to what Waze is doing.

**Reply by Aryan-Sagavekar** [reply]
Yes I found a related issue [#1160](https://github.com/organicmaps/organicmaps/issues/1160).
I'll start researching ways to do it.
Also what do you think, is this a big issue(i mean by time and work)?

**Reply by biodranik** [reply]
Start carefully thinking about it from all angles, and you'll get a bigger picture ;-)

**Comment by ghost** [comment]
That would be great, something like waze but preserving the privacy
